pkgdb/src/buildenv/assets/etc/profile.d/0100_common-paths.sh

```diff
@@ -7,7 +7,7 @@
 PATH="$FLOX_ENV/bin:$FLOX_ENV/sbin${PATH:+:$PATH}"
 FPATH="$FLOX_ENV/share/zsh/vendor-completions${FPATH:+:$FPATH}"
 FPATH="$FLOX_ENV/share/zsh/site-functions:$FPATH"
-MANPATH="$FLOX_ENV/share/man${MANPATH:+:$MANPATH}"
+MANPATH="$FLOX_ENV/share/man${MANPATH:+:$MANPATH}:"
```

Comment:
```suggestion
MANPATH="$FLOX_ENV/share/man:${MANPATH:+:$MANPATH}"
```

@mkenigs

- If MANPATH starts with `:` we now have `::` -> rule 3, the defaults are inserted in between, i.e. in front of MANPATH, but FLOXENV will take precedence in any case 
- If MANPATH ends with `:` we end with `:` -> rule 2, the defaults are appended (no change)
- IF MANPATH doe not start or end with `:`, FLOX_ENV:MANPATH replaces the defaults (no change)